bims/admin.py

```diff
@@ -214,6 +215,21 @@ def update_site_code(self, request, queryset):
             sender=LocationSite
         )
 
+        if queryset.count() > 10:
+            """ Update site code in background"""
+            for location_s in queryset:
```

Comment:
@Fanevanjanahary it's better to process all of them at once in the background. I think you can just send the list of ids to worker.